packages/neuron-wallet/src/utils/ckb-rpc.ts

```diff
@@ -118,19 +118,62 @@ const lightRPCProperties: Record<string, Omit<Parameters<CKBRPC['addMethod']>[0]
   },
 }
 
+class Method extends SdkRpcMethod {
+  constructor(node: CKBComponents.Node, options: CKBComponents.Method) {
+    super(node, options, rpcConfig)
+  }
+}
+
 export class FullCKBRPC extends CKBRPC {
+  constructor(url: string, rpcConfig: Partial<RPCConfig>) {
+    super(url, rpcConfig)
+    this.setNode({ url })
+
+    this.getSyncState = new Method(this.node, {
+      name: 'getSyncState',
+      method: 'sync_state',
+      paramsFormatters: [],
+      resultFormatters: (state: {
+        assume_valid_target: string
+        assume_valid_target_reached: boolean
+        best_known_block_number: any
+        best_known_block_timestamp: any
+        fast_time: any
+        ibd: any
+        inflight_blocks_count: any
+        low_time: any
+        normal_time: any
+        orphan_blocks_count: any
```

Comment:
Type definition of `SyncState` could be found at https://github.com/nervosnetwork/ckb/tree/develop/rpc#type-syncstate

Reply:
Extended this [interface](https://github.com/ckb-js/lumos/blob/develop/packages/rpc/src/types/rpc.ts#L302) will be better